araproc/framework/data_visualization.py

```diff
@@ -117,16 +117,35 @@ def plot_waveform_bundle(
     plt.close(fig)
     del fig, axd
 
-def plot_skymap(the_map = None,
+def plot_skymap(st,list_of_landmarks = None,cal_pulse_index = None,spice_depth = None,the_map = None,
                 output_file_path = None
                 ):
+    """
+    This function returns skymap for given map type
+    Parameters
+    ----------
+    list_of_cal_pulser_indices : list ## example [0,1,2,3]
+       which calpulsers you want to see in your skymap
+    list_of_landmarks: list ## example ['ICL',IC22S']
```

Comment:
Sorry to be a pain, but I know this will be easier for the user: can you list all valid landmark options here?

Reply:
No worries!